### DISCUSSION TITLE
Question regarding S3 assets and which url to use.

### DISCUSSION THREAD

**Discussion by victorronnow** [brefphp/bref#1638] [Support] [answered]
Hi!

I'm pretty new to lambda and php, so hope to get some questions answered here.
I have some assets that get bundled using vite.js inside a `public/dist` directory like this.

## Setup

```
public
├── dist
│   ├── desktop-7d1fa30d.js
│   ├── main-b84a6fd7.css
│   ├── manifest.json
│   └── mobile-9194bc24.js
└── index.php
```

I've installed the `serverless-lift` extension and setup my serverless.yml like this:

``` yaml
constructs:
  website:
    type: server-side-website
    assets:
      '/dist/*': public/dist/
```

Right now, after deploying my app using `serverless deploy` I get this:

``` zsh
~/www/site
❯ serverless deploy

Deploying app to stage dev (us-east-1)

✔ Service deployed to stack app-dev (315s)

endpoint: ANY - https://j6anl9xtbc.execute-api.us-east-1.amazonaws.com
functions:
  app: app-dev-app (1.7 MB)
website:
  url: https://dsdi2l6rwp34qk.cloudfront.net
  cname: dsdi2l6rwp34qk.cloudfront.net

Want a better experience than the AWS console? Try out https://dashboard.bref.sh
```

## The problem:

My app can't access the assets when accessed through this url:
`https://j6ltl9xtxc.execute-api.us-east-1.amazonaws.com/dist/desktop-7d1fa30d.js`

But it works using this cloudfront one:
`https://dsdi2l6rwp34qk.cloudfront.net/dist/desktop-7d1fa30d.js`

_I've obstructed the urls, so you can't test with those urls atm._

Any particular reason for this? Should I be concerned about this or simply use the cloudfront url?

Hope you can help me 😄

**Comment by mnapoli** [comment, marked as answer]
Hi, indeed you should not use the first URL (it's the URL for API Gateway). The second is the right one!

**Reply by victorronnow** [reply]
Thanks Matthieu!
